ts/types/services/ServicePreference.ts

```diff
@@ -0,0 +1,15 @@
+import { IUnitTag } from "italia-ts-commons/lib/units";
+
+/**
+ * The unique ID of a Service
+ */
+export type ServiceID = string & IUnitTag<"ServiceID">;
```

Comment:
```suggestion
export type ServiceID = string & IUnitTag<"ServiceID">;
```
The existing definition is not enough?
`definitions/backend/ServiceId.ts`

Reply:
You're right!